core/src/main/java/tech/tablesaw/api/DateColumn.java

```diff
@@ -97,6 +101,10 @@ public DateColumn(String name, IntArrayList data) {
         super(name);
         this.data = data;
     }
+    
+    public void setLocale(Locale locale) {
+        this.locale = locale;
```

Comment:
I think it would be better to pass this in via the constructor rather than a setter

Reply:
Let me add this to the constructor as well.

Reply:
You can create two constructors with it being optional in one of them. Immutability is almost always better. We should avoid having any setters